solidity/contracts/Chainlinked.sol

```diff
@@ -13,6 +13,13 @@ contract Chainlinked {
   LinkToken internal link;
   Oracle internal oracle;
   uint256 internal requests = 1;
+  mapping(bytes32 => bool) internal unfulfilledRequests;
+
+  event ChainlinkRequested(bytes32 id);
+  event ChainlinkFulfilled(bytes32 id);
+  event ChainlinkCancelled(bytes32 id);
+
+  event ChainlinkRequest(bytes32 id);
```

Comment:
We have `ChainlinkRequest` and `ChainlinkRequested`, while I think the intention was to only have one. `ChainlinkRequested` is currently unused but fits in better with the other event names.